cg/cli/workflow/balsamic/store.py

```diff
@@ -105,6 +115,29 @@ def generate_deliverables_file(context, dry, config_path, case_id):
     process = subprocess.run(" ".join(command), shell=True)
```

Comment:
There is a module in CG for calling shell commands. Please use that so we don't run this kind of code in different ways in different places

Reply:
I agree that this probably go into a BalsamicAPI that uses command.Process, I don't have time for that kind of refactoring ATM, can we push it to the future?

Reply:
Perhaps make an issue?